src/updater.ts

```diff
@@ -209,6 +215,14 @@ export class Updater {
     }
   }
 
+  private async touchLastRun(): Promise<void> {
+    try {
+      await deps.file.touch(this.lastrunfile)
```

Comment:
we should probably just make it so touch does this

Reply:
done